### ISSUE TITLE
keyfinder auto=yes recomputes the keys from scratch on 'beet import' only

### ISSUE THREAD

**Issue by ribeaud** [beetbox/beets#3854]
### Use case

This is rather an improvement request, an inconsistency than a real feature request. Up to the documentation:
> **auto**: Analyze every file on import. Otherwise, you need to use the `beet keyfinder`
> command explicitly.

This is true. However, both possibilities do not work the same way. `beet keyfinder` will try to find the _initial_key_ in the database as well. `beet import` will NOT do so and will start re-computing the keys from scratch if not found in the file itself.

### Solution

Both workflows should work the same. And `beet import` should lookup the db as well.

**Comment by sampsyo** [comment]
Can you clarify whether you're talking about _re-imports_? That is, importing a file that is already in your library? If so, this is a fundamental restriction on re-imports. They start "from scratch," using the metadata that is on the files, ignoring the database—just like a "fresh" import. #3552 has more details.

**Comment by sampsyo** [comment]
The reason I ask is that, unless you're doing a re-import, your files aren't in the database—so it doesn't seem possible the data could already exist in the database, only in the files' tags. Does that make sense?

**Comment by ribeaud** [comment]
Sorry, I am not sure I got it. Let's try it differently.

This afternoon, I ran a `beet import`. All the keys are now in the db. This is for sure and I checked (with `beet info -l`). Running `beet import` is now very quick (because all the keys have already been computed).

If I now execute `beet import <my-music-folder>` (I only have ONE music folder, no new file), then, for files NOT containing an _initial_key_ as part of their written metadata, it will re-compute the _initial_key_ (NOT looking up in the database first).
